pytorch_lightning/core/lightning.py

```diff
@@ -1440,7 +1457,9 @@ def tbptt_split_batch(self, batch, split_size):
         Note:
             Called in the training loop after
             :meth:`~pytorch_lightning.callbacks.base.Callback.on_batch_start`
-            if :paramref:`~pytorch_lightning.trainer.Trainer.truncated_bptt_steps` > 0.
+            if :paramref:`~pytorch_lightning.core.lightning.LightningModule.truncated_bptt_steps` > 0
+            or :paramref:`~pytorch_lightning.trainer.Trainer.truncated_bptt_steps` > 0
```

Comment:
Let's not mention the deprecated option?

```suggestion
            if :paramref:`~pytorch_lightning.core.lightning.LightningModule.truncated_bptt_steps` > 0.
```